pkgs/servers/monitoring/plugins/zfs.nix

```
@@ -0,0 +1,40 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3
+, zfs
+, sudo
+}:
+
+let
+  n="check_zfs";
+in stdenv.mkDerivation {
+  pname = "check-zfs";
+  version = "2.0";
+
+  src = fetchFromGitHub {
+    owner  = "zlacelle";
+    repo   = "nagios_check_zfs_linux";
+    rev    = "33bea0671953a8891b46db6951f56c59a120a16c";
+    sha256 = "gPLCNt6hp4E94s9/PRgsnBN5XXQQ+s2MGcgRFeknXg4=";
```

```suggestion
    owner = "zlacelle";
    repo = "nagios_check_zfs_linux";
    rev = version;
    sha256 = "gPLCNt6hp4E94s9/PRgsnBN5XXQQ+s2MGcgRFeknXg4=";
```

You can use `rec` to make `version` here inheritable.